public/stylesheets/404Style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  background-color: #f2f2f2;
+  margin: 0;
+}
+
+#navbar {
+  padding: 1rem;
+  background-color: #f0dbdb;
+}
+
+#page-title {
+  margin: 0;
+  color: #dba39a;
+  font-size: 1.5rem;
+}
```

Comment:
there's no need to re-do these styles, just add the `style.css` into the html file